data-pipeline/src/pipeline/pre_processing/local_authority.py

```diff
@@ -318,7 +326,7 @@ def _prepare_ons_la_population_data(
     dtypes = input_schemas.la_ons_population.get(
         year, input_schemas.la_ons_population["default"]
     )
-    dtypes[str(year)] = "float"
+    dtypes[str(year - 1)] = "float"
```

Comment:
Pretty sure the answer's yes but for safety's sake: do we always have the year-minus-one data? i.e. the file contains 2020 data for the 2021 run.